client/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:14
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN yarn install --network-concurrency 2
```

Comment:
concurrency reduction was added to have the `install` pass

`npm install` was getting blocked by trussworks
`yarn install` was getting blocked by [lodash](https://github.com/yarnpkg/yarn/issues/6312)